lib/src/widgets/order_booking/summary/order_amount_summary.dart

```diff
@@ -53,9 +53,14 @@ class OrderAmountSummaryWidget extends StatelessWidget {
     required this.totalQty,
     required this.mainColor,
     required this.currencyUtil,
+    required this.qty,
```

Comment:
Make it nullable as it should receive value only when showTotalQtyInAllUnits = true.